lib/commands/on-complete.js

```diff
@@ -1,9 +1,10 @@
 var assert = require('assert');
 var log = require('db-migrate-shared').log;
 
-module.exports = function (migrator, internals, callback, originalErr) {
+module.exports = function (migrator, internals, callback, originalErr, results) {
   if (typeof callback !== 'function') {
     originalErr = originalErr || callback;
+    results = results || originalErr;
```

Comment:
@wzrdtales I removed the change from line 6 as you requested.

That said, I don't understand under what circumstances `callback` is provided yet is not a function. 

I'm concerned about leaving line 6 in its original state because if `callback` is not provided and `results` are, then `originalErr` will contain the results and not an error.

Reply:
LMK if that doesn't make sense to you, not sure if I worded that as clearly as I could have...

Reply:
If there is an error, there will be no results though :)

Reply:
And yeah ok, the callback thing, that would need to fit with the condition if the callback is there or not, which is the only unique circumstance here.

Reply:
But if no callback is passed, that should be valid anyway, since you do not need the results even if they're existent, since you do not deliver them.

Reply:
Hah! That is true. Still seems a bit fragile to me but I'm OK with it if you are :)